tests/test_hook.py

```diff
@@ -25,6 +25,11 @@
 CONFIG_RUNTIME = "nodejs12.x"
 CONFIG_INSECURE_SSL = False
 ACTIVE = True
+TEST_PAYLOAD_TO_HOOK = "{'payload': {'action': 'user_update'}}"
```

Comment:
It seems to me according to the API reference this should actually not be a string but the actual dict (JSON object)?

Reply:
When you are using `api_client.py` directly in Python then that should be a dictionary. But when using a cli, you cannot pass a dictionary, you can only pass the string (similarly to when using curl, as shown[ in our API docs ](https://api.elis.rossum.ai/docs/#test-a-hook)). Given that all these tests test cli commands rather than the direct requests, it made sense to me to use the string as this is the form in which the information will arrive to the command from cli.

Reply:
Sorry for imprecise phrasing - you are right, that makes sense. But what I'm concerned about is that when this is passed to the Rossum API, it should be already in a parsed dict form.

(I should have preceded this by an appropriate suggestion in the actual functional code.)

Reply:
Actually, Rossum API excepts the string itself and parses it itself. Was tested by building the code as it was.

I have updated the logic.

Reply:
wow ok sorry for bringing it up in that case!! (and underestimating our BE team :) )

Thanks for adding the parsing step though, perhaps failing early is more userfriendly.